Huawei/netflow-9.conf

```diff
@@ -1,12 +1,42 @@
+
+slot 1
+ip netstream sampler to slot self
+#
+slot 2
+ipv6 netstream sampler to slot self
+#
 ip netstream timeout active 1
 ip netstream timeout inactive 15
-ip netstream export version 9
-
+ip netstream export version 9 origin-as
+#
 # this command is only necessary if you run VRP >= 5.0
 # it will store interface indices in 32bit counters (as SNMP does) instead of 16bit.
 ip netstream export index-switch 32
-
 ip netstream export template timeout-rate 1
+ip netstream sampler fix-packets {{device_sample_rate}}  inbound
+ip netstream export source {IP-ADDRESS}
+ip netstream export host{{kentik_ingest_IP}} {{kentik_ingest_PORT}} vpn-instance INTERNET
+#
+ipv6 netstream timeout active 1
+ipv6 netstream timeout inactive 15
+ipv6 netstream export version 9 origin-as
+#
+ip netstream monitor FLOW-IPv4
+ip netstream export host {{kentik_ingest_IP}} {{kentik_ingest_PORT}} vpn-instance INTERNET version 9
```

Comment:
either remove `vpn-instance INTERNET`, or rename it to `vpn-instance <vpn-instance>` as that is needed only if they are sending flow traffic over specific VRF instance